qemu/tests/cfg/virtio_fs_localfs_migration.cfg

```diff
@@ -0,0 +1,82 @@
+    fs_source_dir_targetfs = /var/tmp/virtio_fs_target/
+    fs_target = myfs
+    fs_dest = /mnt/${fs_target}
+    force_create_fs_source = no
+    remove_fs_source = no
+    fs_driver_props = {"queue-size": 1024}
+    fs_binary_extra_options = ''
+    driver_name = viofs
+
+    test_file = 'test_file'
+    test_data = 'hello_virtiofs'
+    pre_command = "mkdir -p ${fs_source_dir_fs} ${fs_source_dir_targetfs}"
+    pre_command += " && echo -e ${test_data} > ${fs_source_dir_fs}/${test_file}"
+    pre_command += " && echo -e ${test_data} > ${fs_source_dir_targetfs}/${test_file}"
+    post_command = "rm -rf ${fs_source_dir_fs} ${fs_source_dir_targetfs}"
+    read_file_cmd = "cat %s/${test_file}"
+
+    share_mem = yes
+    vm_mem_share = yes
+    vm_mem_backend = memory-backend-memfd
+    Win10.i386:
+        mem = 4096
+    !s390, s390x:
+        mem_devs = mem1
+        backend_mem_mem1 = memory-backend-memfd
+        size_mem1 = ${mem}M
+        use_mem_mem1 = no
+        guest_numa_nodes = shm0
+        numa_memdev_shm0 = mem-mem1
+        numa_nodeid_shm0 = 0
+    Windows:
+        # install winfsp tool
+        i386, i686:
+            install_winfsp_path = 'C:\Program Files'
+            devcon_dirname = 'x86'
+        x86_64:
+            install_winfsp_path = 'C:\Program Files (x86)'
+            devcon_dirname = 'amd64'
+        install_winfsp_cmd = 'msiexec /i WIN_UTILS:\winfsp.msi /qn'
+        check_installed_cmd = 'dir "%s" |findstr /I winfsp'
+        viofs_log_file = C:\viofs_log.txt
+        viofs_svc_name = VirtioFsSvc
+        viofs_exe_path = C:\virtiofs.exe
+        viofs_exe_copy_cmd = xcopy %s C:\ /Y
+        viofs_sc_create_cmd = 'sc create ${viofs_svc_name} binpath=${viofs_exe_path} start=auto'
+        viofs_sc_create_cmd += ' depend="WinFsp.Launcher/VirtioFsDrv" DisplayName="Virtio FS Service"'
+        viofs_sc_start_cmd = 'sc start ${viofs_svc_name}'
+        viofs_sc_query_cmd = 'sc query ${viofs_svc_name}'
+        viofs_sc_delete_cmd = 'sc delete ${viofs_svc_name}'
+        debug_log_operation = 'enable'
+        viofs_debug_enable_cmd = 'reg add HKLM\Software\VirtIO-FS /v DebugFlags /d 0xFFFFFFFF /t REG_DWORD'
+        viofs_log_enable_cmd = 'reg add HKLM\Software\VirtIO-FS /v DebugLogFile /d ${viofs_log_file} /t REG_SZ'
+        viofs_debug_delete_cmd = 'reg delete HKLM\Software\VirtIO-FS /v DebugFlags /f'
+        viofs_log_delete_cmd = 'reg delete HKLM\Software\VirtIO-FS /v DebugLogFile /f'
+        viofs_reg_query_cmd = 'reg query HKLM\Software\VirtIO-FS'
+        virtio_win_media_type = iso
+        cdroms += " virtio"
+        read_file_cmd = "type %s\${test_file}"
+    variants:
+        - cache_mode_auto:
+            fs_binary_extra_options += " --writeback --cache auto"
+        - cache_mode_always:
+            fs_binary_extra_options += " --writeback --cache always"
+        - cache_mode_never:
+            fs_binary_extra_options += " --allow-direct-io --cache never"
+        - cache_mode_metadata:
```

Comment:
For the cache mode matrix here, we also need to cover other modes( auto, always, never) standalone without `writeback/allow-direct-io`